libbeat/common/kubernetes/watcher.go

```diff
@@ -255,12 +255,17 @@ func (w *watcher) watch() {
 			r := w.k8sResourceFactory()
 			eventType, err := watcher.Next(r)
 			if err != nil {
-				logp.Err("kubernetes: Watching API error %v", err)
 				watcher.Close()
-				if !(err == io.EOF || err == io.ErrUnexpectedEOF) {
+				switch err {
+				case io.EOF:
+					logp.Debug("kubernetes", "EOF while watching API")
+				case io.ErrUnexpectedEOF:
+					logp.Warn("kubernetes: Unexpected EOF while watching API")
```

Comment:
Historically we tried not to use `Warn` but Error or Info to make it clear if action is needed or not. I'm tempted to think this should be logged on `Info` level as no action is needed normally?

Reply:
Yeah, changed to `Info` :+1: